test/extended/util/cli.go

```diff
@@ -176,6 +182,7 @@ func (c *CLI) SetupProject() {
 	newNamespace := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("e2e-test-%s-", c.kubeFramework.BaseName))
 	c.SetNamespace(newNamespace).ChangeUser(fmt.Sprintf("%s-user", newNamespace))
 	e2e.Logf("The user is now %q", c.Username())
+	ProwGCPSetup(c)
```

Comment:
this is starting to seem like a weird place to call this.  i'm going to lgtm it because i don't think it hurts anything but i would think there is a more top level "starting the extended test binary" location that we could setup env vars exactly once.

Reply:
thanks